forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -79,10 +79,10 @@ const PrivateRoute = React.memo((props) => {
   const [kcInstance, setKcInstance] = React.useState(getKcInstance());
 
   const authenticate = (instance, store) => {
-    dispatch(setUserAuth(instance.isAuthenticated()));
     store.dispatch(
       setUserRole(JSON.parse(StorageService.get(StorageService.User.USER_ROLE)))
```

Comment:
here I am doing first dispatching USER_ROLES then dispatching userAuhenticated as true. so in private router page there are somany checks with user_roles. if I dispatch userAuthentication first then it will be check the router permission without role initially